nginx/nginx.conf

```diff
@@ -41,7 +41,9 @@ http {
 	# Remove the Connection header if the client sends it,
 	# it could be "close" to close a keepalive connection
 	proxy_set_header Connection '';
+	## comment me and uncomment below me to support ssl
 	proxy_set_header Host $host;
+	#proxy_set_header Host $http_host;
```

Comment:
Why this?

Reply:
this was based on #550 but i will try rolling it back incase it was another change i didn't need

Reply:
i infact did not need it, but what is #550 all about then? when you just want ssl via HTTP ?

Reply:
@phyzical I think they were using an older version of our Nginx config, which was lacking the Host setting.